### ISSUE TITLE
Can't run price pusher for SUI testnet due to beta price IDs not being found

### ISSUE THREAD

**Issue by web1web1** [pyth-network/pyth-crosschain#2171]
Hi,

I am trying to test out Pyths price feeds for my Sui application. In order to make sure I can test with a regular cadence of price updates, I'm running Pyths price pusher found in this repo using the public Hermes endpoint.

I am using the official testnet object IDs found here: https://docs.pyth.network/price-feeds/contract-addresses/sui (using the Ids from following the links, as the actual text differs from the link its tied to and the text seems to be outdated objects on testnet.

In my asset yaml I have defined SUI/USD using the official Beta price IDs found here: https://www.pyth.network/developers/price-feed-ids#beta

The problem is, when I try to run the pusher (or query Hermes manually via curl) it doesn't seem to have any of the Beta ids, and when I try to use the stable ids, there is no entry in the on chain table for the stable ids.

The stable ids not being present on chain makes sense to me due to the directive to use Beta ids on the price-feed-ids page, however what doesn't make sense is that Hermes doesn't support these ids.

I've gotten your pusher working on other chains (Solana specifically) pretty easily, so I'm relatively confident its not a major skill issue on my side, but please let me know what you think or if there is some way to resolve this.

Thanks in advance for your help.

TL;DR: Hermes doesn't seem to support the Beta Ids so I can't use the pusher on SUI testnet. Pls help.

**Comment by aditya520** [comment]
Which public hermes link are you using? 

Please use `https://hermes-beta.pyth.network/` for beta feeds.

**Comment by web1web1** [comment]
> Which public hermes link are you using?
> 
> Please use `https://hermes-beta.pyth.network/` for beta feeds.

Okay so it was a major skill issue on my part dang. Just so I can figure out where I missed that there was a beta endpoint, do you know that it's documented anywhere?

Thanks so much for your help

**Comment by aditya520** [comment]
It was removed recently from the docs. Thank you for pointing this out, it will be added soon back to the docs.
